pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr failed

pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test.TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20980593?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20980593?buildTab=artifacts#/) on master @ [213837425b4a7e57ea83d072d53a81ee0e010054](https://github.com/cockroachdb/cockroach/commits/213837425b4a7e57ea83d072d53a81ee0e010054):


```
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 6m10.07226205s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 6m20.073297604s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 6m30.081255786s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 6m40.084573402s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 6m50.084762676s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 7m0.085651413s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 7m10.085936489s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 7m20.086771856s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 7m30.08939204s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 7m40.091440199s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 7m50.091631254s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 8m0.092579375s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 8m10.094127543s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 8m20.094685228s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 8m30.095462225s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 8m40.095557785s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 8m50.09563363s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 9m0.09595844s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 9m10.096278984s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 9m20.096393029s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 9m30.097460805s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 9m40.097940407s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 9m50.098028961s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 10m0.098091696s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 10m10.098232981s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 10m20.098423769s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 10m30.098535639s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 10m40.099206573s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 10m50.10012281s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 11m0.100224691s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 11m10.100329119s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 11m20.100531645s
    datadriven.go:357: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19351/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/schemachangerccl/sctestbackupmrccl/sctestbackupmrccl_test_/sctestbackupmrccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/schemachangerccl/testdata/end_to_end/alter_table_alter_primary_key_rbr/alter_table_alter_primary_key_rbr.definition:58: still running after 11m30.100783295s
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ff746806d784a78ced2ec9427a3f2cc4/logTestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr2288532116
--- FAIL: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr (692.68s)
=== RUN   TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec/restore_all_tables_in_database
    backup.go:624: post-RESTORE schema change was expected to not succeed
--- FAIL: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec/restore_all_tables_in_database (25.23s)
=== RUN   TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec/restore_database
    backup.go:624: post-RESTORE schema change was expected to not succeed
--- FAIL: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec/restore_database (23.42s)
=== RUN   TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec/restore_database_with_schema_only
    backup.go:624: post-RESTORE schema change was expected to not succeed
--- FAIL: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec/restore_database_with_schema_only (23.73s)
=== RUN   TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24
    test_server_shim.go:95: cluster virtualization disabled due to issue: #156308 (expected label: C-test-failure)
--- FAIL: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24 (304.44s)
=== RUN   TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec
--- FAIL: TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr/post_commit_stage_24_of_24/post_1_rollback_stage_exec (73.85s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRollbacks_multiregion_alter_table_alter_primary_key_rbr.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58726

--- a/pkg/sql/schemachanger/sctest/backup.go
+++ b/pkg/sql/schemachanger/sctest/backup.go
@@ -467,8 +467,9 @@ func backupRollbacks(
 				url:                url,
 				mayRollback:        true,
 				expectedOnRollback: postRollback,
-				// Speed up mixed version variants by excluding individual table restores.
-				excludeAllTablesInDatabaseFlavor: isMultiRegion && isMixedVersion,
+				// Speed up multi-region or mixed version variants by excluding
+				// individual table restores, due to the slower speed due to multiple nodes.
+				excludeAllTablesInDatabaseFlavor: isMultiRegion || isMixedVersion,
 			}
 			var name string
 			switch i {

